Index: src/components/text-field/text-field.pug
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>include ../text/text\r\n\r\nmixin text-field(args= {withType: null, withLabel:null, withSublabel:null, withName:null, withId:null, withValue:null, withPlaceholder:null, withPattern:null, withFocused: false})\r\n\t-\r\n\t\tvar args = args\r\n\t\tvar inputClass = \"text-field__input\"\r\n\r\n\t\tif (args.withFocused) {\r\n\t\t\tinputClass += ` text-field__input_focused`\r\n\t\t}\r\n\t.text-field\r\n\t\tlabel(for=args.withId, class=\"text-field__label\")\r\n\t\t\t.text-field__label-wrapper\r\n\t\t\t\t+text({\r\n          withTag: \"h3\",\r\n          withContent: args.withLabel\r\n        })\r\n\t\t\t\t+text({\r\n          withTag: \"h3\",\r\n          withContent: args.withSublabel,\r\n          withWeight: \"normal\",\r\n          withColor: \"dark-shade-50\"\r\n        })\r\n\t\tinput(class=inputClass, type=args.withType, name=args.withName, id=args.withId, value=args.withValue, placeholder=args.withPlaceholder)\r\n
===================================================================
diff --git a/src/components/text-field/text-field.pug b/src/components/text-field/text-field.pug
--- a/src/components/text-field/text-field.pug	
+++ b/src/components/text-field/text-field.pug	
@@ -1,11 +1,11 @@
 include ../text/text
 
-mixin text-field(args= {withType: null, withLabel:null, withSublabel:null, withName:null, withId:null, withValue:null, withPlaceholder:null, withPattern:null, withFocused: false})
+mixin text-field(args= {withType: null, withLabel:null, withSublabel:null, withName:null, withId:null, withValue:null, withPlaceholder:null, withPattern:null, withFocus: false})
 	-
 		var args = args
 		var inputClass = "text-field__input"
 
-		if (args.withFocused) {
+		if (args.withFocus) {
 			inputClass += ` text-field__input_focused`
 		}
 	.text-field
